.github/workflows/common-ci.yml

```diff
@@ -0,0 +1,26 @@
+name: Common CI
+
+on:
+  pull_request:
+    paths-ignore:
+      - '.ci/**'
+      - '.github/actions/**'
+      - '.github/workflows/zeebe-*'
+      - 'Dockerfile'
+      - 'bom/*'
+      - 'build-tools/**'
+      - 'clients/**'
+      - 'dist/**'
+      - 'parent/*'
+      - 'pom.xml'
```

Comment:
I'm not a fan of this approach. It feels fragile. Any changes to the folder structure would could break it. 

Do we have no other way to resolve this problem?

How do we deal with this in the `main` branch?

Reply:
Pretty much in the same way 🤣 https://github.com/camunda/zeebe/blob/main/.github/workflows/common-ci.yml